lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -249,9 +249,11 @@ defmodule RealtimeWeb.RealtimeChannel do
            pg_change_params: pg_change_params
          })}
 
-      {:error, error} ->
-        message = "Access token has expired: " <> Helpers.to_log(error)
+      {:error, error} when is_binary(error) ->
+        shutdown_response(socket, error)
 
+      {:error, error} ->
+        message = Helpers.to_log(error)
```

Comment:
just checking if really don't need "Access token has expired: "

Reply:
In this case I'm not sure honestly 🤔

Reply:
added back